pkg/scheduler/framework/plugins/volumebinding/test_utils.go

```diff
@@ -0,0 +1,232 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package volumebinding
+
+import (
+	"fmt"
+
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	pvutil "k8s.io/kubernetes/pkg/controller/volume/persistentvolume/util"
+	"k8s.io/utils/pointer"
+)
+
+type nodeOpt func(node *v1.Node)
```

Comment:
This is something you are adding in this refactoring, right?

Could you use this pattern instead? https://github.com/kubernetes/kubernetes/blob/master/pkg/controller/job/job_controller_test.go#L2957

Reply:
Sure. Fixed.